engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaComponentProvider.scala

```diff
@@ -38,6 +38,10 @@ class LiteKafkaComponentProvider extends ComponentProvider {
     val schemaRegistryTypedJson = "DataSourcesAndSinks#schema-registry--json-serialization"
     val noTypeInfo = "DataSourcesAndSinks#no-type-information--json-serialization"
 
+    val cacheConfig = KafkaConfig.parseConfigOpt(dependencies.config).map(_.schemaRegistryCacheConfig).getOrElse(SchemaRegistryCacheConfig())
```

Comment:
let's use config, not dependencies.config, dependencies.config should be removed in the future probably

Reply:
Be aware that in lite we use kafka inside `modelConfig`. You probable have in mind `FlinkKafkaComponentProvider` where we have `TemporaryKafkaConfigMapping` and indeed we can use component's `config` but ... it will be `config.getConfig("config")` because we have one extra level of nesting. Because of this fact I've decided to leave it consistent with other places where kafka config is used - let's fix all places when we get rid of `TemporaryKafkaConfigMapping`. Maybe we decide to remove this nested level. WDYT?